team-manager/src/main/java/ua/knu/knudev/teammanager/domain/Department.java

```diff
@@ -0,0 +1,36 @@
+package ua.knu.knudev.teammanager.domain;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.UuidGenerator;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(schema = "team_management", name = "department")
+public class Department {
+
+    @Id
+    @UuidGenerator
+    @Column(updatable = false, nullable = false)
+    private UUID id;
+
+    @Column(nullable = false)
+    private String name;
+
+    @ManyToMany(fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
+    @JoinTable(name = "departments_specialties",
+            joinColumns = @JoinColumn(name = "departments"),
+            inverseJoinColumns = @JoinColumn(name = "specialties"))
+    private Set<Specialty> specialties = new HashSet<>();
```

Comment:
@ManyToMany(fetch = FetchType.LAZY)
    @JoinTable(
            name = "departments_specialties",
            schema = "team_management",
            joinColumns = @JoinColumn(name = "department_id"),
            inverseJoinColumns = @JoinColumn(name = "specialtie_id")
    )
    private Set<Specialty> specialties = new HashSet<>();

Make to this, cascade is removed, and added schema name
And also, in `departments_specialties` columns should be deparment_id and speciality_id

Also reflect this changes to liquibase

Reply:
done